django/db/backends/base/features.py

```diff
@@ -302,6 +302,8 @@ class BaseDatabaseFeatures:
     # Does this backend require casting the results of CASE expressions used
     # in UPDATE statements to ensure the expression has the correct type?
     requires_casted_case_in_updates = False
+    # Does the backend support casting a tuple to a text value?
+    can_cast_tuple_to_text = False
```

Comment:
```suggestion
    supports_cast_tuple_to_text = False
```
(nit) I think "supports" is used quite a bit